app-feature-preview/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@color/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_logo"/>
+    <monochrome android:drawable="@drawable/ic_launcher_logo" />
```

Comment:
`@drawable/ic_launcher_logo` is not a monochrome icon.

Reply:
The issue with the current monochrome vector is that it fails for API 21-23. Not shure why not for the legacy project.

Reply:
Lint then complains about a missing monochrome icon

Reply:
Seems fair, we don't have one :smile: Pretending that we do, just to make Lint happy, seems wrong. Let's just suppress the warning for test apps.